src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/iOSSpecific/PopUp.shared.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using XFPC = Xamarin.Forms.PlatformConfiguration;
+using XCTElement = Xamarin.CommunityToolkit.UI.Views.BasePopup;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.iOSSpecific
+{
+	public static class PopUp
+	{
+		public static readonly BindableProperty ArrowDirectionProperty = BindableProperty.Create(
+			"ArrowDirection", typeof(PopoverArrowDirection), typeof(BasePopup), PopoverArrowDirection.None);
+
+		public static void SetArrowDirection(BindableObject element, PopoverArrowDirection color) =>
+			element.SetValue(ArrowDirectionProperty, color);
+
+		public static PopoverArrowDirection GetArrowDirection(BindableObject element) =>
+			(PopoverArrowDirection)element.GetValue(ArrowDirectionProperty);
+
+		public static IPlatformElementConfiguration<XFPC.iOS, XCTElement> UseArrowDirection(this IPlatformElementConfiguration<XFPC.iOS, XCTElement> config, PopoverArrowDirection value)
```

Comment:
need to use the `XFPC.iOS` because the compiler complains that `iOS` is a namespace